src/libs/musicplayer/OpenALMusicPlayer.cpp

```diff
@@ -161,7 +161,7 @@ bool OpenALMusicPlayer::streamBuffer(ALuint buffer)
 {
 	char pcm[BUFFERSIZE];
 	int size = 0;
-	const char* error = '\0';
+	const char* error = "";
```

Comment:
Can you please reason out in PR description how this change provided support for g++. (You can show the error logs as well. )